configs/conda.yml

```diff
@@ -65,3 +67,4 @@ dependencies:
   - bioconductor-scater
   - bioconductor-glmGamPoi
   - r-gh
+  - r-presto
```

Comment:
We had to remove this in the next PR because presto recipe required R >= 4.3 < 4.4 ... although, I had presto working with R 4.3 in my machine... but whatever, presto is optional, to speed up DE testing. We can omit, and we do that with the next PR..